Vagrantfile

```diff
@@ -0,0 +1,53 @@
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
+
+release       = ENV['RELEASE'] ? ENV['RELEASE'] : 'stable'
+hostname      = ENV['HOSTNAME'] ? ENV['HOSTNAME'] : 'st2vagrant'
+private_ip    = ENV['PRIVATE_IP'] ? ENV['PRIVATE_IP'] : '192.168.16.20'
```

Comment:
We definitely don't need so many confgured vars, leave only those which are 100% needed